src/_data/basic-project.json

```diff
@@ -0,0 +1,6 @@
+{
+	"url": "https://github.com/MarcoMicale/starting-project",
+	"name": "Basic Project",
+	"description": "A single page template using 11ty e vanilla CSS.",
```

Comment:
Is the "e" here intentional? To me it feels like it should be an "and".

Reply:
yes